go.mod

```diff
@@ -4,6 +4,9 @@ go 1.20
 
 require (
 	github.com/Khan/genqlient v0.6.0
+	github.com/hashicorp/terraform-plugin-framework v1.3.1
```

Comment:
I have a feeling the changes to go.mod and go.sum are the cause of the unrelated failing tests. Maybe some other pieces need changes with these changes to dependencies. And that is out of scope for this ticket. I've rebuilt `main` here https://buildkite.com/buildkite/terraform-provider-buildkite-main/builds/574 and it passes just fine. So I'm advocating for reverting these changes to see if it helps